### PR TITLE
RT-677-hide-remove-layer-button:Hide remove layer button in cardio app

### PR DESCRIPTION
**Pull request by yashpmohod**
## Overview
Hide the remove layer button in the cardio-health app.
Use the Input decorator to set hidden true.
 
image

## How it was tested

Tested using run showcase,cardio-health, and cardio-patient app locally.

## Checklist

- [x] The title contains a short meaningful summary
- [x] I have added a link to this PR in the Jira issue
- [x] I have described how this was tested
- [x] I have included screen shots for changes that affect the user interface
- [ ] I have updated unit tests
- [x] I have run unit tests locally
- [ ] I have updated documentation (including README)

### CODE REVIEW

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.html [hunk 1]
```diff
@@ -35,7 +35,7 @@
       </mat-expansion-panel-header>
       <data-layer-options [layer]="layer" (layerChange)="onLayerChange($event)"></data-layer-options>
       <mat-action-row>
-        <button mat-button color="primary" (click)="onLayerRemove(layer)" title="Remove this layer"><mat-icon fontIcon="delete"></mat-icon> REMOVE LAYER</button>
+        <button mat-button color="primary" (click)="onLayerRemove(layer)" title="Remove this layer" [hidden]="showRemoveButton"><mat-icon fontIcon="delete"></mat-icon> REMOVE LAYER</button>
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.html:38]: It looks like `showRemoveButton` is doing the opposite of what it says

**Review by semicolin — CHANGES_REQUESTED** [review]

File: projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.html (hunk 1 above)
Comment (yashpmohod) [reply projects/ngx-charts-on-fhir/src/lib/data-layer-list/data-layer-list/data-layer-list.component.html]: okay @semicolin

File: projects/ngx-charts-on-fhir/src/lib/fhir-chart-layout/fhir-chart-layout.component.html [hunk 2]
```diff
@@ -5,7 +5,7 @@
       <div class="layout-sidebar-content">
         <patient-browser *ngIf="toolbar.includes('patients')" [hidden]="active !== 'patients'"></patient-browser>
         <data-layer-browser *ngIf="toolbar.includes('browser')" [hidden]="active !== 'browser'"></data-layer-browser>
-        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'"></data-layer-list>
+        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'" [hideRemoveLayerButton]="hideRemoveLayerButton"></data-layer-list>
```
Comment (semicolin) [inline projects/ngx-charts-on-fhir/src/lib/fhir-chart-layout/fhir-chart-layout.component.html:8]: fhir-chart-layout can decide to hide the button based on whether the browser panel is available:
```suggestion
        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'" [hideRemoveLayerButton]="!toolbar.includes('browser')"></data-layer-list>
```
That way the button is always hidden when there's no way to add layers and the app doesn't have to configure the button explicitly.

**Review by semicolin — CHANGES_REQUESTED** [review]
I had another idea for how to simplify this...

**Review by amolht — APPROVED** [review]

**Pull request merged**